test/test_distribution.cpp

```diff
@@ -143,6 +143,16 @@ TEST(MxxDistribution, DistributeVector) {
 
     test_distribute<std::vector<int>>(size, gen, c);
     test_stable_distribute<std::vector<int>>(size, gen, c);
+
+    // create a distribution of total size smaller than
+    // the total number of processes and zero elements on the last process
+    size = (c.rank() % 2 == 0) ? 1 : 0;
+    if (c.is_last()) {
+      size = 0;
+    }
+    // XXX: test_distribute fails with an assertion error in mxx::sort
+    // test_distribute<std::vector<int>>(size, gen, c);
```

Comment:
This seems to fail with an assertion error in sample_distribute. Is that expected?